docs/static/releasenotes.asciidoc

```diff
@@ -67,6 +68,33 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-15-3]]
+=== Logstash 8.15.3 Release Notes
+
+[[notable-8.15.3]]
+==== Notable issues fixed
+
+* Improved the pipeline bootstrap error logs, it now includes the cause's backtrace giving a hint where the issue occurred https://github.com/elastic/logstash/pull/16495[#16495]
+
+* Fixed Logstash core compatibility issues with `logstash-input-azure_event_hubs` previous versions `<1.4.8` https://github.com/elastic/logstash/pull/16485[#16485]
```

Comment:
The standard for version info is to say "versions 1.4.8 and earlier," but since `1.4.8` is in backticks and rendered as literal, I'm not sure that's the right thing to do here. I trust your expertise on this one. Whatever you decide is fine.

Reply:
I prefer the standard way, thanks for reviewing, Karen! :)

Reply:
```suggestion
* Fixed Logstash core compatibility issues with `logstash-input-azure_event_hubs` versions `1.4.8` and earlier https://github.com/elastic/logstash/pull/16485[#16485]
```